templates/web/tfl/report/new/councils_text_all.html

```diff
@@ -0,0 +1,16 @@
+<p>
+[% UNLESS non_public_categories.$category;
+
+    tprintf(
+        'Information you enter here will be sent to us and published for others to see, in accordance with our <a href="%s">privacy policy</a>.',
```

Comment:
I think this means the `fixmystreet.update_public_councils_text` override in `tfl/js.js` can be dropped now?